src/components/modal/ModalDialog.js

```diff
@@ -49,6 +65,24 @@ class ModalDialog extends React.Component<Props> {
 
     modalID: string = uniqueId('modal');
 
+    /**
+     * Renders a button if onBackButtonClick is passed in
+     * @return {ReactElement|null} - Returns the button, or null if the button shouldn't be rendered
+     */
+    renderBackButton() {
```

Comment:
I'm wondering if design would settle for a `Back` button in the  `ModalActions` footer rather than introducing a new pattern that we don't use anywhere else.